### ISSUE TITLE
All directives except leaflet are removed in favor of lf-directiveName [Breaking Change]

### ISSUE THREAD

**Issue by adgoncal** [tombatossals/angular-leaflet-directive#1005]
TLDR; 
"controls" is a boolean attribute of the HTML5 video element and I think the controls directive should be namespaced to lf-controls, similar to what was done with lf-center (#914).

Long story:
I just ran into an issue where my custom controls would show up in development, but not in production.
I am minifying my html for production using [jonschlinkert/gulp-htmlmin](https://github.com/jonschlinkert/gulp-htmlmin), which uses [kangax/html-minifier](https://github.com/kangax/html-minifier).
I had the option "collapseBooleanAttributes" set to true, so when the html was minified the controls attribute had its value omitted. 

This happens because the HTML5 video element has a boolean attribute called controls and the html-minifier library is using a very general regular expression to check for boolean attributes. One may argue that this is an issue with html-minifier, but I still think it would be good to namespace the controls directive.

**Comment by nmccready** [comment]
I do not think this needs to be requested as an individual directive. All of them need to be renamed to lf-whatever. The only directive to stay the same would be leaflet itself.

This is a backlog ticket that got lost in #535

**Comment by tombatossals** [comment]
I'm going to rework&redesign angular-leaflet-directive to be compatible with Leaflet v1.0. It will mantain almost all its functionality, and will be compatible with the current features of the directive, but I must start from a fresh point, so I'm going to close this issue. If you think it must be worked with the new version, please reopen it.
